harness/determined/cli/experiment.py

```diff
@@ -195,6 +195,21 @@ def create(args: Namespace) -> None:
         submit_experiment(args)
 
 
+def limit_offset_paginator(
+    method: Callable, agg_field: str, connection_args: Namespace, **kwargs: Any
+) -> List[Union[bindings.v1Experiment, bindings.trialv1Trial]]:
+    all_objects: List[Union[bindings.v1Experiment, bindings.trialv1Trial]] = []
+    offset = 0
+    while True:
+        r = method(setup_session(connection_args), limit=200, offset=offset, **kwargs)
```

Comment:
I don't think we should setup session for every page separately. 
in my snippet I was just passing `setup_session(...)` as an opaque `*args` parameter. this also has a benefit of taking a session from everywhere, e.g. in python sdk where you don't have `connection_args` from argparse.